saltproc/tests/integration_tests/const_repr/test_const_reproc_run.py

```diff
@@ -9,8 +9,8 @@
 path = os.path.realpath(__file__)
 sys.path.append(os.path.dirname(os.path.dirname(path)))
 directory = os.path.dirname(path)
-db_exp_file = directory+'/2step_non_ideal_2.h5'
-db_file = directory+'/db_saltproc.h5'
+db_exp_file = directory + '/2step_non_ideal_2.h5'
```

Comment:
How come this file doesn't also have `/data`?

Reply:
Because it's pregenerated and is stored in `const_repr` directory.